x/ecocredit/go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/gorilla/mux v1.8.0
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
 	github.com/pkg/errors v0.9.1
+	github.com/pariz/gountries v0.0.0-20200430155801-1c6a393df9c7
```

Comment:
Why is this needed?

Reply:
Not using anywhere, removing it.

Reply:
This was from the retirement location stuff, but should have been removed - perhaps this snuck in a merge/rebase at some point, then disappeared after squash/merge.